lib/tapioca/compilers/sorbet.rb

```diff
@@ -26,7 +26,9 @@ def run(*args)
 
         sig { returns(String) }
         def sorbet_path
-          SORBET.to_s.shellescape
+          sorbet_path = ENV.fetch("TAPIOCA_SORBET_EXE", SORBET)
```

Comment:
Apologies for seeing this late, but I think we can do one more small refactor by pulling this env var name out to a constant on this class. We can refer to the env var in tests by using the constant as well. 

That way, if we ever decide to change the name or something, we would only need to do it in one place.

Reply:
@paracycle done! Apologies for finishing this on a weekend, I'm in no rush on this change; you can re-review on Monday. Thank you for the thorough code review – I learned a couple of new things about Ruby from you! 🙂

Reply:
Amazing! Great to hear that and thanks again for the contribution.